gobblin-metrics-libs/gobblin-metrics-base/src/main/java/org/apache/gobblin/metrics/event/TimingEvent.java

```diff
@@ -109,21 +122,75 @@ public void stop() {
    *
    * @param additionalMetadata a {@link Map} of additional metadata that should be submitted along with this event
    */
+  @Deprecated
```

Comment:
Add alternative way, the same applied to `stop()`

Reply:
done